Update the UA Reduction Origin Trial Blog Post (#1572)
GoogleChrome/developer.chrome.com: site/en/blog/user-agent-reduction-origin-trial/index.md

1.  See [https://uar-ot.glitch.me/](https://uar-ot.glitch.me/) for a simple
    demonstration of the origin trial (along with the source code).

> ## How do I participate in the origin trial as a third-party embed?

Reviewer: ```suggestion
## How to participate in the origin trial as a third-party embed?
```

> Starting in Chrome 96, third-party embeds (e.g. an iframe inside another site)
> can participate in the origin trial without requiring the top-level site to be
> enrolled.

Reviewer: ```suggestion
Starting in Chrome 96, third-party embeds (for example, an iframe inside another site)
can participate in the origin trial without requiring the top-level site to be
enrolled.
```
